### ISSUE TITLE
Comment on examples

### ISSUE THREAD

**Issue by MKatz-DHSC** [best-practice-and-impact/qa-of-code-guidance#62]
Probably a good idea to add comments to your example code saying if it is demonstrating good or bad practice

**Comment by foster999** [comment]
Agreed, I've noticed that the first example under "Comments" is bad but this isn't clear unless you read the text.
